From 4a754cfb9795bfd9f1f818141386b6a34d2f1185 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Sun, 7 Apr 2013 18:55:12 +0800
Subject: [PATCH 444/635] ENGR00257141-2 fixed external display UI flicker

Surfaceflinger does not unblank external display,
The initialized status should be set as unblank.

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 mx6/hwcomposer/hwc_display.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mx6/hwcomposer/hwc_display.cpp b/mx6/hwcomposer/hwc_display.cpp
index c639d92..687cf30 100644
--- a/mx6/hwcomposer/hwc_display.cpp
+++ b/mx6/hwcomposer/hwc_display.cpp
@@ -188,7 +188,7 @@ int hwc_get_framebuffer_info(displayInfo *pInfo)
     pInfo->xdpi = 1000 * (info.xres * 25.4f) / info.width;
     pInfo->ydpi = 1000 * (info.yres * 25.4f) / info.height;
     pInfo->vsync_period  = 1000000000 / refreshRate;
-    pInfo->blank  = 1;
+    pInfo->blank  = 0;
 
     ALOGV("using\n"
           "xres         = %d px\n"
-- 
1.8.0

